weave-js/src/components/PagePanelComponents/Home/Browse3/filters/FilterBar.tsx

```diff
@@ -30,7 +30,7 @@ import {combineRangeFilters, getNextFilterId} from './filterUtils';
 import {GroupedOption, SelectFieldOption} from './SelectField';
 import {VariableChildrenDisplay} from './VariableChildrenDisplayer';
 
-const DEBOUNCE_MS = 700;
+const DEBOUNCE_MS = 1_000;
```

Comment:
Is the implication here that if for some reason a state update takes longer than 1 second (as opposed to 700ms) the old state may still appear?

It seems like there were some logic changes in this PR but it's not immediately apparent to me what the debounce vs the filter state changes address respectively

Reply:
updated the pr description for clarity on the functional changes.